index.js

```diff
@@ -212,14 +238,6 @@ const checkEmail = () => {
 form.addEventListener('submit', (e) => {
   if (checkEmail() === false) {
     e.preventDefault();
-  } else {
-    const data = {
-      name: namefield.value,
-      email: email.value,
-      text: textarea.value,
-    };
-
-    localStorage.setItem('data', JSON.stringify(data));
   }
 });
```

Comment:
- [ ] Good job so far, As requested by the previous reviewer please make sure the form data is saved in the local storage once the form is filled , you can use the _**change eventlistener**_ to set the form data to localstorage and check if there is formdata on the localstorage the form is populated with that data when the page reloads, you can use the _**reload or domcontentloaded**_ event for this, currently no data is being saved to local storage, see below :point_down: 

![image](https://user-images.githubusercontent.com/97521557/202840014-f15c86a9-6281-4ef6-a3d9-d1a4eb6fd30d.png)

How it should be when the form is filled :point_down: 

![image](https://user-images.githubusercontent.com/97521557/202840033-ac1b7384-908d-40ea-b743-035a9ddb2743.png)


- [ ] [OPTIONAL] I think it would be better if you create a separate js file for this functionality like a localStorage.js file :+1: